docs/guidelines.md

```diff
@@ -0,0 +1,43 @@
+# Plugin Author Guidelines
+
+Netlify plugins listed in the plugins.json file of this repository can be installed directly within the Netlify UI from the plugins directory. To help ensure a consistent user experience, we've prepared the following guidelines for authors submitting plugins to the directory.
+
+Before submitting a plugin for the plugins directory, please read and follow the guidelines below.
+
+## Provide a zero-config default.
+
+Plugins with required `inputs` cannot be installed via the Netlify UI. Wherever possible, a plugin should include default options that allow the plugin to run without configuration. In cases where a plugin requires a unique value (such as an API key for a third-party service), configure the plugin to accept this value from a [build environment variable](https://docs.netlify.com/configure-builds/environment-variables). For more complex or customized configuration, users can install the plugin via the `netlify.toml` configuration file.
+
+## Provide a README.
+
+Every plugin in the Netlify plugins directory includes a link to the plugin README. This file should include:
+- A description of what the plugin does and why that might be useful.
+- Any required environment variables.
```

Comment:
Also optional ones?

Reply:
Hm, good point. I want the required env vars to be separate, though, so it's clear they're needed for _all_ installations. I can add the optional ones to the next point.